earth_enterprise/src/fusion/portableglobe/gerewritedbroot.cpp

```diff
@@ -260,26 +275,27 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
           "           --kml_port=<kml_port_string> \\\n"
           "           --kml_url_path=<kml_url_path_string> \\\n"
           "           --use_ssl_for_kml=<use_ssl_for_kml_bool> \\\n"
+          "           --preserve_kml_filenames \\\n"
           "\n"
           " Reads dbroot and rewrites the search tabs so that they point\n"
-          " at the given search server and port. Creates a directory\n"
-          " of all of the icons referred to by the dbroot. Creates a file.\n"
-          " listing all kml files that are referenced in the dbroot and.\n"
-          " gives their new name on the kml server.\n"
+          " to the given search server and port. Optionally creates a directory\n"
+          " of all of the icons referred to by the dbroot. \n"
+          " Optionally updates the host, path and/or filename to the \n"
+          " network linked KML content in dbRoot. \n"
```

Comment:
I think it's confusing to be referring to network-linked KML content.  I don't think there's any processing of KML, or network links done.  The `gerewritedbroot` utility is just changing the URLs of the KML wrapper layers.